backend/v2/epochs/schemas.py

```diff
@@ -0,0 +1,29 @@
+from pydantic import BaseModel
+
+
+class CurrentEpochResponse(BaseModel):
+    current_epoch: int
```

Comment:
We should align if we want to have a V1 suffix for all the api objects that are 1:1 with Flask

Reply:
kk